src/components/Intercom/Intercom.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState, useCallback } from 'react'
+import IntercomWidget from 'decentraland-dapps/dist/components/Intercom'
+import { getAnalytics, getAnonymousId } from 'decentraland-dapps/dist/modules/analytics/utils'
+import { config } from 'config'
+
+const APP_ID = config.get('INTERCOM_APP_ID', '')
+
+export const Intercom: React.FC = () => {
+  const [dclAnonymousUserID, setDclAnonymousUserID] = useState('')
+
+  const analytics = getAnalytics()
```

Comment:
Is analytics always going to be defined?

Reply:
yep, analytics is always going to be defined